{
  "cve_id": "CVE-2022-0273",
  "github_data": {
    "repository": "janeczku/calibre-web",
    "fix_commit": "0c0313f375bed7b035c8c0482bbb09599e16bfcf",
    "related_commits": [
      "0c0313f375bed7b035c8c0482bbb09599e16bfcf",
      "0c0313f375bed7b035c8c0482bbb09599e16bfcf"
    ],
    "patch_url": null
  },
  "vulnerability_details": {
    "cvss_score": 6.5,
    "cvss_vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:H/A:N",
    "cwe_id": "CWE-284",
    "description": "Improper Access Control in Pypi calibreweb prior to 0.6.16.",
    "attack_vector": "NETWORK",
    "attack_complexity": "LOW"
  },
  "temporal_data": {
    "published_date": "2022-01-30T14:15:07.597",
    "last_modified": "2024-11-21T06:38:16.947",
    "fix_date": null
  },
  "references": [
    {
      "url": "https://github.com/janeczku/calibre-web/commit/0c0313f375bed7b035c8c0482bbb09599e16bfcf",
      "source": "security@huntr.dev",
      "tags": [
        "Patch",
        "Third Party Advisory"
      ]
    },
    {
      "url": "https://huntr.dev/bounties/8f27686f-d698-4ab6-8ef0-899125792f13",
      "source": "security@huntr.dev",
      "tags": [
        "Exploit",
        "Issue Tracking",
        "Patch",
        "Third Party Advisory"
      ]
    },
    {
      "url": "https://github.com/janeczku/calibre-web/commit/0c0313f375bed7b035c8c0482bbb09599e16bfcf",
      "source": "af854a3a-2127-422b-91ae-364da2661108",
      "tags": [
        "Patch",
        "Third Party Advisory"
      ]
    },
    {
      "url": "https://huntr.dev/bounties/8f27686f-d698-4ab6-8ef0-899125792f13",
      "source": "af854a3a-2127-422b-91ae-364da2661108",
      "tags": [
        "Exploit",
        "Issue Tracking",
        "Patch",
        "Third Party Advisory"
      ]
    }
  ],
  "collection_metadata": {
    "collected_at": "2025-01-11T23:02:57.095465",
    "processing_status": "raw"
  }
}